common/src/main/java/dev/latvian/kubejs/block/BlockBuilder.java

```diff
@@ -209,6 +214,16 @@ public BlockBuilder jumpFactor(float f) {
 		return this;
 	}
 
+	/**
+	 * Parameters for callback are BlockContainerJS and java.util.Random.
+	 *
+	 * @param randomTickCallback
+	 */
+	public BlockBuilder randomTick(@Nullable BiConsumer<BlockContainerJS, Random> randomTickCallback) {
```

Comment:
As Lat said on Discord, this should be an event object to stay consistent with KubeJS' overall callback style